### ISSUE TITLE
Terminal: Path that contain spaces are not linked properly

### ISSUE THREAD

**Issue by moi90** [microsoft/vscode#196753]
Does this issue occur when all extensions are disabled?: (Can't check because I'm working remotely.)

Related to #97941.

#169786's goal was to correctly implement links with spaces for lines that only contain a path. However, that does not work for me.

Versions:
```
Version: 1.70.2
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:36:25.715Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.15.0-87-generic
```

The relevant contents of my terminal are:
```
$ find "$PWD" -maxdepth 5 -name "meta.yaml.bak"

/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/PS122-4_49-16/Haul 1/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/0046_PS122-4_46-90_24hours-4/Haul 5/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/0046_PS122-4_46-80_24hours-1/Haul 1/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/PS122-4_46-64/Haul 3/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/PS122-4_46-64/Haul 2/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/PS122-4_46-64/Haul 1/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/0046_PS122-4_46-89_24hours-3/Haul 3/LOKI_10001.01/meta.yaml.bak
/projects/loki/_raw_data/PS122 (MOSAiC 2019_2020)/Leg4/PS122-4_48-17/Haul 1/LOKI_10001.01/meta.yaml.bak
```

Instead of underlining each whole line with a path, the individual segments separated by a space are underlined.

**Comment by meganrogge** [comment]
You're on a really old version, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
/label info-needed

**Comment by Tyriar** [comment]
Yes that change actually landed in v1.75, let me know if it happens on v1.83+

**Comment by moi90** [comment]
It doesn't happen on 1.83.1

I'm sorry, I should have checked that I run the latest version. I just assumed it, but seemingly, automatic updates have been disabled when I upgraded my Ubuntu.
